lib/farscape/plugins.rb

```diff
@@ -34,9 +34,9 @@ def disabled_plugins
     # Else if :default_state was passed in return :default_state
     def why_disabled(options)
       maybe = @disabling_rules.map { |hash| hash.select { |k,v| k if v == options[k] } }
-      maybe |= disabled_plugins.map { |k,v| v[:name] }
+      maybe |= [disabled_plugins[:name]]
```

Comment:
Yes, in fact this would just have made everything fail if something was named `:name`

:P